src/busctl/busctl.c

```diff
@@ -2503,7 +2507,8 @@ static int help(void) {
                "     --watch-bind=BOOL     Wait for bus AF_UNIX socket to be bound in the file\n"
                "                           system\n"
                "     --destination=SERVICE Destination service of a signal\n"
-               "     --num-matches=NUMBER  Exit after receiving a number of matches while\n"
+               "  -N --limit-messages=NUMBER\n"
+               "                           Exit after receiving a number of matches while\n"
```

Comment:
"matches" -> "messages"

Reply:
Also, maybe rephrase this as "Stop monitoring after receiving a specified number of messages"?